lxd/api_cluster.go

```diff
@@ -2911,8 +2914,75 @@ func clusterNodeStatePost(d *Daemon, r *http.Request) response.Response {
 		return response.BadRequest(err)
 	}
 
+	stopFunc := func(inst instance.Instance) error {
```

Comment:
Please can you define these function inside the `if req.Action == "evacuate" {` section to limit their scope. 
This indicates to the reader quickly that they are only relevant for evacuation.